### PR TITLE
New1

### PR DESCRIPTION
**Pull request by sareenasaji**

### CODE REVIEW

File: .firebase/hosting.cHVibGlj.cache
Comment (ajayesivan) [inline .firebase/hosting.cHVibGlj.cache:1]: We don't need this file in git

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-.vscode/
\ No newline at end of file
+.vscode/
```
Comment (ajayesivan) [inline .gitignore]: Add the following code to `.gitignore` file

```gitignore
# Logs
logs
*.log
npm-debug.log*
yarn-debug.log*
yarn-error.log*
firebase-debug.log*
firebase-debug.*.log*

# Firebase cache
.firebase/

# Firebase config

# Uncomment this if you'd like others to create their own Firebase project.
# For a team working on the same Firebase project(s), it is recommended to leave
# it commented so all members can deploy to the same project(s) in .firebaserc.
# .firebaserc

# Runtime data
pids
*.pid
*.seed
*.pid.lock

# Directory for instrumented libs generated by jscoverage/JSCover
lib-cov

# Coverage directory used by tools like istanbul
coverage

# nyc test coverage
.nyc_output

# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
.grunt

# Bower dependency directory (https://bower.io/)
bower_components

# node-waf configuration
.lock-wscript

# Compiled binary addons (http://nodejs.org/api/addons.html)
build/Release

# Dependency directories
node_modules/

# Optional npm cache directory
.npm

# Optional eslint cache
.eslintcache

# Optional REPL history
.node_repl_history

# Output of 'npm pack'
*.tgz

# Yarn Integrity file
.yarn-integrity

# dotenv environment variables file
.env
```

**Review by ajayesivan — CHANGES_REQUESTED** [review]

File: .firebase/hosting.cHVibGlj.cache
Comment (sareenasaji) [reply .firebase/hosting.cHVibGlj.cache:1]: Ok I will update👍🏻.

File: .gitignore (hunk 1 above)
Comment (sareenasaji) [reply .gitignore]: 👍🏻

**Comment by Adhithyarajan** [comment]
@sareenasaji Please try to give commit messages mentioning the changes you've created on respective PRs.

**Review by ajayesivan — APPROVED** [review]
@sareenasaji please merge this PR.

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<title>Document</title>
+<link rel="stylesheet" href="style.css">
+<script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+<h3 class="text-3xl text-green-400 font-bold underline">Hey it's me Sareena Saji
+</h3>
+
+
+<script src="script.js" type="text/javascript"></script>
```
Comment (ajayesivan) [inline public/index.html]: Wrap this in`<html></html>`.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+<link rel="stylesheet" href="style.css">
+<script src="https://cdn.tailwindcss.com"></script>
+</head>
```
Comment (ajayesivan) [inline public/index.html:6]: This `head` tag doesn't have a opening pair

**Review by ajayesivan — CHANGES_REQUESTED** [review]
Also, update the PR name.

File: public/index.html (hunk 3 above)
Comment (sareenasaji) [reply public/index.html:6]: yes all corrected.

**Review by ajayesivan — APPROVED** [review]
You can merge this

**Pull request merged**